task/github-add-comment/0.7/tests/fixtures/github-post-comment.yaml

```diff
@@ -0,0 +1,35 @@
+# Default GitHub
+---
+headers:
+  method: PATCH
+  path: /repos/{repo:[^/]+/[^/]+}/issues/comments/{comment:[0-9]+}
+response:
+  status: 200
+  output: '{"status": 200}'
+  content-type: text/json
+---
+headers:
+  method: GET
+  path: /repos/{repo:[^/]+/[^/]+}/issues/{issue:[0-9]+}/comments
+response:
+  status: 200
+  content-type: text/json
+  file: /fixtures/list-comment-response.json
```

Comment:
do we still need this?

Reply:
Yes, because we need the mock api for the default behaviour (without prefix) and with a prefix (e.g. GH Enterprise). If we deleted this there would not be a possibility to check if the task parameter actually works.